assets/js/components/AdminMenuTooltip/AdminMenuTooltip.js

```diff
@@ -44,7 +44,13 @@ export function AdminMenuTooltip( { onDismiss, tooltipStateKey, ...props } ) {
 		tooltipStateKey,
 	] );
 
-	return <JoyrideTooltip onDismiss={ handleDismissTooltip } { ...props } />;
+	// Render in a Portal to avoid adding an element in the sub-header which has
+	// added styles if not empty. This also avoids a tiny shift when dismissing.
+	return (
+		<Portal slug="ga4-activation-banner-admin-menu-tooltip">
+			<JoyrideTooltip onDismiss={ handleDismissTooltip } { ...props } />
```

Comment:
This should probably be done inside `JoyrideTooltip` instead but I kept it here to minimize the changes. We can follow up with this to be done in all cases internally in a new issue.

Reply:
Opened
- https://github.com/google/site-kit-wp/issues/6049